manifests/p/Panic/Playdate/1.1.0/Panic.Playdate.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Panic.Playdate

```

```suggestion
PackageIdentifier: Panic.Mirror
```

- Please update file name and path to match this change.